app/assets/javascripts/darkswarm/services/map_configuration.js.coffee

```diff
@@ -4,6 +4,7 @@ Darkswarm.factory "MapConfiguration", ->
       center:
         latitude: -37.4713077
         longitude: 144.7851531
+      cluster_icon: 'assets/map_009-cluster.svg'
```

Comment:
I see we're mixing two approaches here. It seems that until now we were fetching from https://github.com/openfoodfoundation/openfoodnetwork/blob/65433c6ac6cef70b201b85b556e943b31825f92d/app/serializers/api/cached_enterprise_serializer.rb#L120-L129. To me that's a client-side responsibility so I wouldn't mind leaning towards having these paths in here. What are your thoughts @Matt-Yorkley @luisramos0 @mkllnk ?

What brought to this line however is something else. I'd like to find a way to use Rails' [image_tag](https://apidock.com/rails/v3.2.13/ActionView/Helpers/AssetTagHelper/image_tag) so we can leverage Rails' [asset hashing](https://guides.rubyonrails.org/v3.2/asset_pipeline.html#in-production). We're suffering many performance issues and there are tons of wasted CPU cycles requesting assets that never change.

Reply:
@sauloperez thanks for pointing this out, I am still finding my way through the code. Any suggestions on how to proceed? I feel that I should be putting all in one place and then we resolve the icons not being processed through rails asset pipeline

Reply:
It's fine. Let's investigate this in a separate PR. Your changes are pretty good and I want them in production :joy: Thanks for considering it though!